src/index.html

```diff
@@ -11,7 +11,148 @@
+        <legend>Personal information</legend>
+
+        <label class="fields">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            maxlength="8"
+          >
+        </label>
+
+        <label class="fields">
+          Name:
+          <input
+          type="text"
+          name="userName"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="fields">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+
+        <label class="fields">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            max="1979-12-31"
+            min="2000-01-02"
+          >
+        </label>
+        
+        <label class="fields">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="wrapper">
+        <legend> Registration</legend>
+        <label class="fields">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="fields">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="wrapper">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="fields">
+          Do you love cats?
+          <input type="radio"  name="radio">
+          Yes
+          <input  type="radio" name="radio">
+           No
+        </label>
+
+        <label class="fields">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="fields">
+          What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+
+        <label class="fields">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="fields">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+          >
+          </input>
+        </label>
+      </fieldset>
+
+      <fieldset class="wrapper">
+        <legend>Additional info:</legend>
+
+        <label class="fields">
+          Comments:
+          <textarea
+            name="feedBack"
+            cols="10"
+            rows="2">
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="feedBack"
            cols="10"
            rows="2"
          ></textarea>
```
Let's fix this problem
![image](https://user-images.githubusercontent.com/96409563/220445863-b9d6ad4a-558b-444c-bb1a-007ad4c598ea.png)
![image](https://user-images.githubusercontent.com/96409563/220445916-bfa60bc4-0c3f-4852-a3b6-06c217fab8fe.png)